### DISCUSSION TITLE
冷知識:未熟牛排流出紅色液體不是血水,是肌肉中蛋白造成的

### DISCUSSION THREAD

**Discussion by 111426027** [1101-GS4719-C/Python-Community-GS4719-C#1726] [Ideas]
肉的顏色主要受兩個因素影響,一是血液中的色素顏色,另一種則是肌肉中的色素。沒有煎熟的肉會呈現出紅色汁液,其實不是血的顏色,而是肌肉中肌紅蛋白(Myoglobin)和水分產生的狀況。因為動物經過屠宰死亡後,在血管中的血紅素(Hemoglobin)經過放血過程而流失,因此影響肉類的顏色呈現以存在於肌肉中的肌紅蛋白的色素為主·根據美國肉類出口協會的資料指出,在放血良好的肌肉組織中,肌紅蛋白較血紅素多而約估肌肉中總色素量的80~90%。因此,生肉顏色呈現的來源大多是肌紅蛋白所影響。煎肉時,肉類中的肌紅蛋白因為加熱熟成作用,會轉變顏色,蛋白質產生變化,而血紅素中的鐵被氧化,因此肉的顏色會轉為灰褐色。但當肉還未熟時,肌肉組織中的肌紅蛋白則會保留有較深的紅色,和周圍的水分、肉汁等產生類似汁液狀態,因此讓人誤為沒熟的肉流出的是血水,其實主要是肌紅蛋白和水分等融合呈現出帶紅色顏色的情況。

補充:肌紅白的含量會因為動物種類、年齡、活動力狀況以及肌肉部位而造成差異,會影響肉質顏色,若肌肉中的肌紅蛋白含量高,肉質便會偏於紅潤,而肌紅蛋白量較低,則肉質顏色較白。像是牛肉、羊肉以及豬肉等肌紅蛋白多,在肉分類中屬於紅肉,而雞肉、魚肉等的肌紅蛋白較低,則被歸類為白肉這也是煎肉過程中,牛肉等流出的肉汁偏紅色,而雞肉等流出的汁液則較偏透明的原因。

資料來源:https://www.greenconut.com/trivia/373_hemoglobin-beef-steak-blood/

看完這篇文章終於知道沒有熟的牛排流出來的紅色液體，原來不是血水，而是蛋白質造成的。
但其實食物當前只會在乎牛排美不美味而已(笑

**Comment by Mitochondrionjessica** [comment]
當肉塊還未熟時，肌肉組織中的肌紅蛋白則會保留有較深的紅色，和周圍的水分、肉汁等產生類似汁液狀態，因此讓人誤以為沒熟的肉流出的是「血水」，其實主要是「肌紅蛋白」和水分等融合呈現出帶紅色顏色的情況。我覺得基本上可以把它理解成組織液體，但吃牛排的時候我比較在意新鮮程度而非熟度。

**Comment by YOGA012** [comment]
作為一位牛排愛好者，我覺得了解肉的顏色來源非常有趣。每次看到牛排流出的紅色汁液時，知道那是肌紅蛋白和水分的結果，而不是血水，讓我對這道美食有了更多的認識。牛肉因為肌紅蛋白含量高，所以顏色紅潤、味道濃郁，這也是為什麼煎牛排時肉汁看起來是紅色的。這些知識讓我在享用美味牛排的同時，也能更加欣賞其中的科學原理。牛排的風味和口感真是無與倫比，每一次品嚐都是一次美妙的享受。

**Comment by 710388** [comment]
哈哈我知道，原本會害怕但是在知道不是血以後就不會怕了

**Comment by NcuJerryKao** [comment]
非常感謝你分享這麼詳細且有趣的資訊！你對肉類顏色的解釋非常準確。讓我總結一下：

肌紅蛋白和血紅素：
肉的顏色主要受兩個因素影響：肌紅蛋白（Myoglobin）和血紅素（Hemoglobin）。
當動物被屠宰後，血管中的血紅素流失，影響了肉類的顏色。因此，生肉的顏色主要來自肌紅蛋白。
煎肉時，肌紅蛋白因加熱而轉變顏色，蛋白質也產生變化，血紅素中的鐵被氧化，使肉的顏色變為灰褐色。
肌紅蛋白含量和肉質顏色：
不同動物種類、年齡、活動力狀況和肌肉部位的肌紅蛋白含量會影響肉質顏色。
牛肉、羊肉和豬肉等紅肉中的肌紅蛋白較多，因此呈現紅潤的顏色。
雞肉、魚肉等白肉中的肌紅蛋白較低，因此呈現較白的顏色。
煎肉過程中的汁液顏色：
煎牛肉等流出的肉汁偏紅色，而雞肉等流出的汁液則較透明。
最後，你說得對，食物的美味才是最重要的！如果牛排好吃，那就是最棒的了，不是嗎？😄

**Comment by cassy0201** [comment]
又吸收到ㄧ個知識了！一直以為動物跟人類一樣有傷口時流出來的是血液，原來其實是肌紅蛋白和水的混合體！

**Comment by tshsu04224** [comment]
原來！現在才知道原來牛排的「血水」其實不是血水，以前吃牛排的時候常常因為怕這些紅色液體，所以不敢點太生的肉，不過這樣咀嚼的時候就要比較費力（？看完這篇文章後，以後會試著嘗試看看比較生的牛排的，長知識了💡

**Comment by 155Julia** [comment]
太扯了，我一直以為那是血水。原來牛排中流出的紅色液體並不是血，而是蛋白質和水分的混合物。這讓我重新認識了牛排的烹飪過程和肉質的特性。了解這些知識後，感覺吃牛排時更加有趣，也更能欣賞到其中的細節。希望以後在享受美食的同時，能有更多這樣的發現和驚喜。

**Comment by yubo0oo** [comment]
肉的顏色主要受兩個因素影響，一是血液中的色素顏色，另一種則是肌肉中的色素。沒有煎熟的肉會呈現出紅色汁液，其實不是血的顏色，而是肌肉中肌紅蛋白和水分產生的狀況。因為動物經過屠宰死亡後，在血管中的血紅素經過放血過程而流失，因此影響肉類的顏色呈現以存在於肌肉中的肌紅蛋白的色素為主。根據美國肉類出口協會的資料指出，在放血良好的肌肉組織中，肌紅蛋白較血紅素多而約佔肌肉中總色素量的80～90%。也因此，此時生肉顏色呈現的來源大多是肌紅蛋白所影響。
